qhub/schema.py

```diff
@@ -487,6 +487,61 @@ def enabled_must_have_fields(cls, values):
 letter_dash_underscore_pydantic = pydantic.constr(regex=namestr_regex)
 
 
+def project_name_convention(value: typing.Any, values):
+    convention = """
+    In order to successfully deploy QHub, there are some project naming conventions which need
```

Comment:
All the other instances refer to Nebari while this refers to QHub

Reply:
This is an existing function that was just moved for practical reasons... updated.